simulator/src/main/java/com/hedera/block/simulator/generator/BlockAsDirBlockStreamManager.java

```diff
@@ -16,7 +16,7 @@
 
 package com.hedera.block.simulator.generator;
 
-import static com.hedera.block.simulator.generator.Utils.readFileBytes;
+import static com.hedera.block.common.utils.FileUtilities.readFileBytesUnsafe;
```

Comment:
Generally, prefer to _not_ static import methods.  It costs little to write `FileUtilities.readFileBytesUnsafe` and is _much_ easier to read and understand without having to simultaneously retain a ton of static imports in your head.

Reply:
✅ Done.